scripts/install_deepchem_conda.sh

```diff
@@ -38,3 +38,9 @@ conda install -y -q -c conda-forge zlib=1.2.11
 conda install -y -q -c conda-forge requests=2.18.4
 conda install -y -q -c conda-forge xgboost=0.6a2
 conda install -y -q -c rdkit rdkit=2017.09.1
+pushd .
+cd $HOME
+git clone https://github.com/deepchem/simdna
```

Comment:
We should install a pinned version from our DeepChem channel.  Do you have a preferred git hash?

Reply:
Good suggestion, fixed.